synapseclient/client.py

```diff
@@ -2358,22 +2359,24 @@ def _getUserbyPrincipalIdOrName(self, principalId: str = None):
                 "Unknown Synapse user (%s).  %s." % (principalId, supplementalMessage)
             )
 
-    @tracer.start_as_current_span("Synapse::getPermissions")
-    def getPermissions(
+    @tracer.start_as_current_span("Synapse::get_acl")
+    def get_acl(
```

Comment:
One additional change you'll need to make for documentation. On [the Client reference page](https://python-docs.synapse.org/reference/client/). I set it up so that we are specifying which functions are documented on the page so that we had control over the order without needing to move all functions in this `client.py`. 

Because of this the new changes you added here also need to be updated in this file: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/docs/reference/client.md?plain=1#L22